### PR TITLE
Adding button to allow submit of new job under jobs

### LINKED ISSUE #60: Job Board: link under top menu to Submit a Job
Add a link under top Job Board menu to Submit a Job -- link could go either to the google form directly, or to a landing page that lists the posting policy and then has a link to the google form.

### PR DESCRIPTION
**Pull request by vsoch**
Signed-off-by: vsoch <vsoch@users.noreply.github.com>

Screenshot from 2021-08-10 13-19-41

Links to (externally, meaning opening in a new tab)

https://docs.google.com/forms/d/e/1FAIpQLSfYK64R1c0rj-ERldGLxuqedLIbsYPZXj9uBplDRYNmnND10Q/viewform

and will close #60

### CODE REVIEW

**Comment by cosden** [comment]
I think I'd prefer if we linked to this page https://vsoch.github.io/usrse.github.io/jobs/policy/ instead of going straight to the form. If it looks like too much text, maybe we could have a big button at the bottom that said "Submit a job"? 

Alternatively, if we left it as you have here, I could copy the text from the policy page to the top of the form....I just think less people will read it. And I want people to read it because this is to proactively answer questions we get on a regular basis.

File: _data/navigation.yml [hunk 1]
```diff
@@ -42,6 +42,9 @@
   dropdown:
     - name: Browse the board
       link: jobs
+    - name: Submit a job
+      link: https://docs.google.com/forms/d/e/1FAIpQLSfYK64R1c0rj-ERldGLxuqedLIbsYPZXj9uBplDRYNmnND10Q/viewform?usp=sf_link
```
Comment (cosden) [inline _data/navigation.yml:46]:
```suggestion
      link: jobs/policy/
```

**Comment by vsoch** [comment]
I actually like the second idea best - going directly to the form and then having another link that goes to policy from there, but I'm okay if this is your preference. Generally if someone wants to submit a job, they don't want the extra clicks (unless they are absolutely interested).

**Comment by vsoch** [comment]
Actually just pushed a better idea - present the reader with BOTH options off the bat so they can read about policy or go directly to submit. And @cosden if it's not there, I'd add a link back to the policy page from the form (or put some of it there directly).

**Comment by cosden** [comment]
+1 - I agree on putting it in the form. I'll do it now. Because it's not live yet, I'll put content on the form rather than the link. When it's live I may change the form to link to the page so it's only in one place if it needs to change.

**Review by cosden — APPROVED** [review]
Just previewed locally. It's perfect, good idea @vsoch!

**Pull request merged**